feature/gribi/ate_tests/weighted_balancing_test/setup_test.go

```diff
@@ -193,11 +193,6 @@ func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
 			t.Fatalf("No address found for port %v", dp)
 		}
 	}
-	if *deviations.ExplicitInterfaceInDefaultVRF {
-		for _, dp := range dut.Ports() {
```

Comment:
Line 196-200 are expected to delete?

Reply:
same as https://github.com/openconfig/featureprofiles/pull/939#discussion_r1062980594

Reply:
SG